typing/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "typing"
```

Comment:
Let's call this crate `air-types` to stick with the convention for other crates.